### PR TITLE
KNL-1452 - MBeanExporter JMX beans not being initialized

### PR DESCRIPTION
**Pull request by jonespm**

### CODE REVIEW

**Comment by jonespm** [comment]
This just removes the NoisierDefaultListableBeanFactory that's out of date and uses the DefaultListableBeanFactory in spring.

**Comment by ern** [comment]
LGTM

File: kernel/kernel-impl/src/test/java/log4j.properties [hunk 1]
```diff
@@ -10,7 +10,7 @@ log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=%d{ABSOLUTE}  %5p %c{1}:%L - %m%n
 
 ### Show debug information for the test harness itself, and any critical collaborators
-log4j.logger.org.sakaiproject.util.NoisierDefaultListableBeanFactory=error
+#log4j.logger.org.sakaiproject.util.NoisierDefaultListableBeanFactory=error
```
Comment (buckett) [inline kernel/kernel-impl/src/test/java/log4j.properties]: Just drop this line?

**Comment by jonespm** [comment]
Okay, removed the commented line

**Comment by ottenhoff** [comment]
ready for merge?

**Comment by jonespm** [comment]
Sure, merging, not too much more we'll see until it's running. The only thing this other one seemed there for was assistance with developers when beans were wired up incorrectly or missing but the error blobs look the same either way to me! ;)

**Pull request merged**